ipinfo/ipinfo_client.go

```diff
@@ -19,7 +19,8 @@ func prepareIpinfoClient(tok string) *ipinfo.Client {
 
 	// attempt to init cache; don't force require it, though.
 	var cache *ipinfo.Cache
-	if !fNoCache {
+
```

Comment:
Think we can remove this extra newline, as the comment above it as well as the variable are strongly connected to the following logic.